es-design-system/static/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "",
+    "short_name": "",
+    "icons": [
+        {
+            "src": "/android-chrome-192x192.png",
+            "sizes": "192x192",
+            "type": "image/png"
+        },
+        {
+            "src": "/android-chrome-512x512.png",
+            "sizes": "512x512",
+            "type": "image/png"
+        }
+    ],
+    "theme_color": "#ffffff",
```

Comment:
Why not use `$primary` here?

Reply:
Will that work in the static-directory?

Reply:
No but you can hard code it.

Reply:
The white-background `#fff` was in consultation with Design. Similarly `#b95100` was recommended over the design-system orange color`#ffbf57` or it's associated gradients. But we can def revisit this.

I think the generated content from [realfavicongenerator](https://realfavicongenerator.net/) covers everything the *Icon Module* from `nuxt/pwa` would cover. But I wonder if the [static/site.webmanifest](https://github.com/EnergySage/es-ds/pull/328/files#diff-6f0ce75410a8767cc13157437c1c31ff65ba5ccbfa66d167ffc80645b7c9a573) would interfear with verticals using [Web App Manifest](https://pwa.nuxtjs.org/manifest) 🤔